lib/nosedrum/storage.ex

```diff
@@ -92,13 +92,27 @@ defmodule Nosedrum.Storage do
   ## Return value
   Returns `:ok` if successful, and `{:error, reason}` otherwise.
   """
-  @callback add_command(
+  @callback queue_command(
```

Comment:
We should update the documentation for this to read that it adds the command to the internal dispatch storage, but does not yet send it to Discord, and also that `process_queued_commands` should be used once all desired commands have been added.

Reply:
Nope, just updating the doc comment would be enough :slightly_smiling_face: